src/sphinx_autodoc_typehints/__init__.py

```diff
@@ -89,6 +89,22 @@ def get_annotation_args(annotation: Any, module: str, class_name: str) -> tuple[
     return getattr(annotation, "__args__", ())
 
 
+def format_internal_tuple(t: tuple, config: Config) -> str:
+    # An annotation can be a tuple, e.g., for nptyping:
+    #   NDArray[(typing.Any, ...), Float]
+    # In this case, format_annotation receives:
+    #   (typing.Any, Ellipsis)
+    # This solution should hopefully be general for *any* type that allows
+    # tuples in annotations
```

Comment:
Please align it alongside 120 characters

Reply:
I'm not sure what you mean -- do you want me to remove the indentation?

Reply:
yes; this line could reach all the way to 120 character, so no need to break it so soon

Reply:
Fixed in aa53976